packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.tsx

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect } from 'react';
+import { FtlMsg } from 'fxa-react/dist/lib/utils';
```

Comment:
I think this is the only instance of an `FtlMsg` import with the `…/dist/…` path, `'fxa-react/dist/lib/utils'`:
```sh
git grep -n "'fxa-react/dist/lib/utils';" | cat
packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.tsx:6:import { FtlMsg } from 'fxa-react/dist/lib/utils';
```

&hellip; although we _do_ have about 54 instances of the non-/dist/ path, `'fxa-react/lib/utils'`:
```sh
git grep -h "import { FtlMsg } from 'fxa-react/lib/utils';" | wc -l # 54
```